integrations/snyk/snyk/client.py

```diff
@@ -290,19 +301,67 @@ async def enrich_project(self, project: dict[str, Any]) -> dict[str, Any]:
         return project
 
     async def create_webhooks_if_not_exists(self) -> None:
-        snyk_webhook_url = f"{self.api_url}/org/{self.org_id}/webhooks"
-        all_subscriptions = await self._send_api_request(
-            url=snyk_webhook_url, method="GET"
-        )
+        all_organizations = await self._get_organizations_in_group()
+        for org_id in all_organizations:
+            logger.info(f"Fetching webhooks for organization: {org_id}")
 
-        app_host_webhook_url = f"{self.app_host}/integration/webhook"
+            snyk_webhook_url = f"{self.api_url}/org/{org_id}/webhooks"
+            all_subscriptions = await self._send_api_request(
+                url=snyk_webhook_url, method="GET"
+            )
 
-        for webhook in all_subscriptions["results"]:
-            if webhook["url"] == app_host_webhook_url:
-                return
+            app_host_webhook_url = f"{self.app_host}/integration/webhook"
 
-        body = {"url": app_host_webhook_url, "secret": self.webhook_secret}
+            for webhook in all_subscriptions["results"]:
+                if webhook["url"] == app_host_webhook_url:
+                    return
 
-        await self._send_api_request(
-            url=snyk_webhook_url, method="POST", json_data=body
-        )
+            body = {"url": app_host_webhook_url, "secret": self.webhook_secret}
+            logger.info(f"Creating webhook subscription for organization: {org_id}")
+            await self._send_api_request(
+                url=snyk_webhook_url, method="POST", json_data=body
+            )
+
+    async def _get_organizations_in_group(self) -> list[Any]:
+        # Check if the result is already cached
```

Comment:
this function supports bringing orgs from multiple groups, while the method name relates to one group, lets adjust the naming